File: 백준/Bronze/2576. 홀수/홀수.c
#include <stdio.h>

int main() {
  int a;
  int sum = 0;
  int max = 101;

  for (int i = 0; i < 7; i++) {
    scanf("%d", &a);
    if (a % 2 != 0) {
      sum += a;
      if (a < max) {
        max = a;
      }
    }
  }

  if (sum == 0) {
      printf("-1");
  }
  else {
  printf("%d\n%d", sum, max);
  }
  return 0;
}

File: 백준/Bronze/2576. 홀수/홀수.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int a, sum = 0, max = 100;
        for (int i = 0; i < 7; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine(), " ");
            a = Integer.parseInt(st.nextToken());
            if (a % 2 == 1) {
                sum += a;
                if (a < max) {
                    max = a;
                }
            }
        }
        if (sum == 0) {
            bw.write("-1");
        } else {
            bw.write(String.valueOf(sum));
            bw.newLine();
            bw.write(String.valueOf(max));
        }
        bw.flush();
        br.close();
        bw.close();
    }
}
